Sources/UI/Web View/WebViewController.swift

```diff
@@ -136,6 +148,15 @@ extension UIViewController {
         let webViewController = WebViewController()
         webViewController.url = url
         webViewController.title = title
-        present(UINavigationController(rootViewController: webViewController), animated: animated)
+        present(WebViewNavigationController(rootViewController: webViewController), animated: animated)
+    }
+}
+
+fileprivate class WebViewNavigationController: UINavigationController {
```

Comment:
It's your custom class. You can make own `init` to avoid searching for the `WebViewController` inside `present` method below.